app/build.gradle

```diff
@@ -24,10 +24,10 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:21.0.3'
-    compile 'com.android.support:support-v4:21.0.3'
+    compile 'com.android.support:appcompat-v7:22.1.1'
+    compile 'com.android.support:support-v4:22.1.1'
```

Comment:
ActionBarActivities are deprecated now and we probably should use AppCompatActivities now.

haven't noticed anything else yet, so you can probably pull this seperately from the pull request